From 24212148223f8772ee97981623f9557dea60c7aa Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Mon, 14 Jul 2014 09:11:36 -0400
Subject: [PATCH] aufs: bool option dependency needs to be "=y".

Otherwise we will see this error for EXPORTFS=m

$make -j40 > /dev/null
fs/built-in.o: In function `aufs_encode_fh':
/home/paul/treegen/tree-pub/fs/aufs/export.c:762: undefined reference to `exportfs_encode_fh'
fs/built-in.o: In function `decode_by_path':
/home/paul/treegen/tree-pub/fs/aufs/export.c:528: undefined reference to `exportfs_decode_fh'
make: *** [vmlinux] Error 1

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>

diff --git a/fs/aufs/Kconfig b/fs/aufs/Kconfig
index 2951f9b3a4fc..4b05b51995bb 100644
--- a/fs/aufs/Kconfig
+++ b/fs/aufs/Kconfig
@@ -71,7 +71,7 @@ endchoice
 
 config AUFS_EXPORT
 	bool "NFS-exportable aufs"
-	depends on EXPORTFS
+	depends on EXPORTFS=y
 	help
 	If you want to export your mounted aufs via NFS, then enable this
 	option. There are several requirements for this configuration.
-- 
2.1.2

